src/components/Banner/Banner.tsx

```diff
@@ -113,8 +100,46 @@ export const Banner = ({
 }: BannerProps) => {
   const platform = usePlatform();
 
+  const HeaderTypography = size === "m" ? Title : Headline;
   const SubheaderTypography = size === "m" ? Text : Subhead;
 
+  const Icon24DismissIOS =
```

Comment:
Может переименовать в

```suggestion
  const IconDismissIOS =
```

чтобы не путаться с иконками из `@vkontakte/icons`

Reply:
@SevereCloud, ➕
Лайкни пулл еще раз, пожалуйста. 😉